include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -3635,6 +3636,218 @@ struct OrtApi {
+  *
+  * The config_key formats are defined in onnxruntime_session_options_config_keys.h
+  *
+  * \param[in] options The session options.
+  * \param[in] config_key A null-terminated UTF-8 string representation of the configuration key.
+  * \param[out] out Pointer set to 1 if the entry exists and 0 otherwise.
+  *
+  * \snippet{doc} snippets.dox OrtStatus Return Value
+  * \since Version 1.14
+  */
+  ORT_API2_STATUS(HasSessionConfigEntry, _In_ const OrtSessionOptions* options,
+                  _In_z_ const char* config_key, _Out_ int* out);
+
+  /** \brief Get a session configuration value.
+  *
+  * The config_key and the format of config_value are defined in onnxruntime_session_options_config_keys.h.
+  * This API can be used to get configuration values for custom operators.
+  *
+  * Returns a failure status if the configuration key does not exist.
+  * The config_key and the format of config_value are defined in onnxruntime_session_options_config_keys.h
+  *
+  * If `config_value` is nullptr, the value of `size` is set to the true size of the string
+  * value (including null-terminator), and a success status is returned.
+  *
+  * If the `size` parameter is greater than or equal to the actual string value's size,
+  * the value of `size` is set to the true size of the string value, the provided memory
+  * is filled with the value's contents, and a success status is returned.
+  *
+  * If the `size` parameter is less than the actual string value's size and `config_value`
+  * is not nullptr, the value of `size` is set to the true size of the string value
+  * and a failure status is returned.
+  *
+  * \param[in] options The session options.
+  * \param[in] config_key A null-terminated UTF-8 string representation of the config key
+  * \param[in] config_value Pointer to memory where the null-terminated UTF-8 string value will be stored.
+  * \param[in,out] size Pointer to the size of the `config_value` buffer. See above comments for details.
+  *
+  * \snippet{doc} snippets.dox OrtStatus Return Value
+  * \since Version 1.14
+  */
+  ORT_API2_STATUS(GetSessionConfigEntry, _In_ const OrtSessionOptions* options,
+                  _In_z_ const char* config_key, _Out_writes_z_(size) char* config_value, _Inout_ size_t* size);
+
+  /** \brief Register custom ops from a shared library.
+  *
+  * Loads a shared library (dll on windows, so on linux, etc) named 'library_path' and looks for this entry point:
+  *		OrtStatus* RegisterCustomOps(OrtSessionOptions * options, const OrtApiBase* api);
+  * It then passes in the provided session options to this function along with the api base.
+  *
+  * The handle to the loaded library is automatically released by ORT when the last OrtSession that references the
+  * library handle is released. If no OrtSession is created, then the library handle is released when the provided
+  * OrtSessionOptions is released.
+  *
+  * \param[in] options
+  * \param[in] library_path
```

Comment:
>library_path [](http://example.com/codeflow?start=15&length=12)

library_path needs clarification. 

For example, I only use the library name without any extension in my current code which uses search path.

It may also mean directory path. Perhaps, we need to say library name which can be fully qualified or relative path depending on the operating system lookup rules?

Reply:
Changed to library_name and added documentation for OS-specific details